Task4.java

```diff
@@ -0,0 +1,15 @@
+package homework3;
+
+// Составьте программу, вычисляющую A*B, не пользуясь операцией умножения.
+
+public class Task4 {
+    public static void main(String[] args) {
+        int a = 5;
+        int b = 4;
+        int c=a;

```

Comment:
Some surplus complication of the logic.
        int result = 0;
        for (int i = 0; i < b; i++) {
            result = result + a;
        }
Please use i,j,k as counters (j in case you have counter in counter, and k in case you have 2 counters inside). It's community predefined standards . Or give a counter specific name.